design-documents/0006-metadata-api/text.md

```diff
@@ -0,0 +1,128 @@
+# Defining Communication between the Portal and the Explorer
+
+## Metadata API
```

Comment:
I don't think we should name this the _Metadata API_ because it is suggests that the API is doing more heavy lifting that it actually does. 

I don't have a specific suggestion for a new name.